src/librustc_data_structures/indexed_set.rs

```diff
@@ -210,6 +210,16 @@ impl<T: Idx> IdxSet<T> {
         self.bits.set_bit(elem.index())
     }
 
+    /// Adds `elem` to the set `self` if `member` is `true`, otherwise removes `elem` from the set;
+    /// returns true iff this changed `self`.
```

Comment:
Do you need this? Alternatively, you have `BitVector` (yes, we ended up with 2 APIs for the same purpose).

Reply:
Yeah, it's used. I don't see how it's immediately similar to `BitVector`...